modules/issue_tracker/php/module.class.inc

```diff
@@ -93,6 +93,41 @@ class Module extends \Module
                     "issue_tracker"
                 )
             ];
+        case 'candidate':
+            $factory = \NDB_Factory::singleton();
+            $DB      = $factory->database();
+            $baseURL = $factory->settings()->getBaseURL();
```

Comment:
These 3 lines are repeated in `usertasks` and `candidate`, whether it is better to first check if the `$type` is in 'usertasks` and `candidate`, if yes, then get the `$factory`, etc. to avoid repeat of the same code?

Reply:
done in the last commit